etl/sde/__init__.py

```
@@ -0,0 +1,5 @@
+"""ETL SDE package for SDE database operations."""
+
+from . import load_sde
+
+__all__ = ['load_sde']
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)